apps/application-system/api/src/environments/environment.ts

```diff
@@ -25,4 +25,5 @@ export default {
   auth: {
     jwtSecret: 'supersecret',
   },
+  fsS3Bucket: process.env.FILE_SERVICE_PRESIGN_BUCKET
```

Comment:
Do you hard code the value for development or pick it up from the process @sindrig?